style.css

```diff
@@ -49,25 +51,65 @@ body {
     white-space: nowrap;
   }
 
-  .nav_menu {
-    display: none;
+  nav {
+    width: 100%;
   }
 
-  .open_mobile_menu_icon {
+  .navMenu {
+    margin-left: -1000px;
+    transition: 0.5s;
+  }
+
+  #navMenu.mobileMenu {
+    display: flex;
```

Comment:
- [ ] [OPTIONAL] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.
- Kindly note that DRY is a requirement, but since the other reviewers have been overlooking it, I've decided to mark it as optional. 😉